dmas/README.md

```diff
@@ -0,0 +1,38 @@
+# DMAS - Data Management and Analysis System
+
+This system will be responsible for the following:
+* Ingesting raw image data into the database
+* Analysing raw image data to extract plant data
+* Matching similar plants and assigning a `plant_id` to them
+* Tracking the growth of plants over time
+
+## Running without docker
+
+```
+source venv/bin/activate
+pip install -r requirements.txt
+uvicorn dmas.app.main:app --reload
+```
+
+## Running with docker
+```
+docker build -t dmas_image ./dmas
```

Comment:
This line implies that these commands should be run in the root of the repo, but this README is within the `dmas` folder. Worth a correction?